File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.py
def solution(n,a,b):
    l, r = min(a, b), max(a,b)
    round = 1
    while True:
        if r % 2 == 0 and l + 1 == r:
            break
        l, r = (l + 1) // 2, (r + 1) // 2
        round += 1
    return round

File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.java
public class Solution {
    public int solution(int n, int a, int b) {
        int r = 0;
        while (a != b) {
            a = (a + 1) / 2;
            b = (b + 1) / 2;
            r++;
        }
        return r;
    }
}